src/components/Footer/Footer.scss

```
@@ -0,0 +1,32 @@
+
+footer {
+  position: relative;
+  bottom: 0;
+  margin-top: 40px;
+  width: 100%;
+  height: auto;
+  background-color: #000;
+}
+
+.footer-nav-list {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 40px;
+  min-width: 800px;
+  max-width: 1300px;
+}
+
+.footer-nav-list li {
+  margin: 10px 0;
+}
```

물론 저도 다시 수정해야 하지만, 풋터 네브 리스트 안에 li 와 a 같은 경우 네브 리스트와 같이 묶어서 표현해 주는 것이 가독성 측면에서 좋다고 합니다. 그래서 묶어서 표현해 주셨으면 합니다. 인덴트가 제대로 안 먹혀서 구조적으로는 아래와 같습니다. 
```suggestion
. footer-nav-list{
        display: flex;
  justify-content: space-between;
  margin: 0 auto;
  padding: 40px;
  min-width: 800px;
  max-width: 1300px;
  
  li{
  }
  a{
  }
  }
```